vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/dcapi/request/Oid4vpDCAPIRequest.kt

```diff
@@ -0,0 +1,45 @@
+package at.asitplus.wallet.lib.dcapi.request
+
+import at.asitplus.catching
+import at.asitplus.openid.OpenIdConstants.DC_API_OID4VP_PROTOCOL_IDENTIFIER
+import at.asitplus.wallet.lib.data.vckJsonSerializer
+import at.asitplus.wallet.lib.oidvci.OAuth2Exception.InvalidRequest
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Oid4vpDCAPIRequest(
+    // openid4vp-v<version>-<request-type>
+    val protocol: String,
+    val request: String,
+    val credentialId: Int,
+    val callingPackageName: String,
+    val callingOrigin: String
```

Comment:
Are those member names needed for de-/serialization? Then we should add `@SerialName`